src/main.rs

```diff
@@ -21,6 +21,7 @@ const PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 const PKG_DESCRIPTION: &str = env!("CARGO_PKG_DESCRIPTION");
 
 const QUERY_PATH: &str = "/dns-query";
+const PROXY_PATH: &str = "/proxy";
```

Comment:
Why not let the config file do this? This isn't a .well-known so no reason to expect servers to have the same path.

Reply:
Maybe a future patch? Trying to keep them consistent for now with the go implementation